src/katello/rhsm.py

```diff
@@ -0,0 +1,41 @@
+import sys
+sys.path.append('/usr/share/rhsm')
+
+try:
+    from subscription_manager import action_client
+except ImportError:
+    from subscription_manager import certmgr
+
+try:
+    from subscription_manager.identity import ConsumerIdentity
+except ImportError:
+    from subscription_manager.certlib import ConsumerIdentity
+
+try:
+    from subscription_manager.injectioninit import init_dep_injection
+    init_dep_injection()
+except ImportError:
+    pass
+
+from rhsm.connection import UEPConnection
+
+def lookup_consumer_id():
+    try:
+        certificate = ConsumerIdentity.read()
+        return certificate.getConsumerId()
+    except IOError:
+        return None
+
+def get_uep():
+    key = ConsumerIdentity.keypath()
+    cert = ConsumerIdentity.certpath()
+    uep = UEPConnection(key_file=key, cert_file=cert)
+    return uep
+
+def get_manager():
+    if 'subscription_manager.action_client' in sys.modules:
+        mgr = action_client.ActionClient()
+    else:
+        # for compatability with subscription-manager > =1.13
```

Comment:
nit: looks like a space was misplaced in ```>=``` (and I know you're just moving it to another file).